specs/git/remove_local_branches_not_in_remote.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Remove all local branches that aren't in remote
+command: |-
+  "git branch --merged >/tmp/merged-branches && \
+   vi /tmp/merged-branches && \
+   xargs git branch -d </tmp/merged-branches"
+tags:
+  - git
+description: "Remove all local branches that doesn't exists in remote repository."
```

Comment:
minor grammar nit: I think this should be "Remove all local branches that don't exist in a remote repository."